samples/21.customization-plain-ui/README.md

```diff
@@ -0,0 +1,157 @@
+# Customizing Web Chat with a plain HTML UI
+
+This sample shows how to use Web Chat without its canned UI component.
+
+When we designed Web Chat, we heavily considered the importance of customization. To make customization great and accessible, we designed our components in layers. Developers can get the full experience of Web Chat by:
+
+1. Using all of our layers: using all of our layers as-is,
+2. Using just business layers: building their own UI from the ground up using Web Chat's business logic, or
+3. Using business layers and partly UI components: opting into our UI but replacing just a handful of components as needed.
+
+In this sample, we are demonstrating the ability to rebuild Web Chat UI using just the business layer of Web Chat.
+
+Below is the explanation of different layers of Web Chat.
+
+<table>
```

Comment:
I feel like this could be super beneficial to new users in the root README.md. Should we consider adding a link to this page so people can learn more about the components?

Reply:
Great idea.

But since our sample is scattered (or littered) in `SAMPLES.md`, instead of copying to `SAMPLES.md` for now. I would prefer cleaning up `SAMPLES.md` or building a GitHub Wiki out of it.